backend/apps/events/resolvers.py

```diff
@@ -77,16 +77,7 @@ def resolve_default_events(self, info):
         """
         For each organization, get the most recent (future) event
         """
-        organizations = Organization.objects.all()
-        all_events = Event.objects.filter(start_time__gte=date.today())
-        events = []
-        for organization in organizations:
-            try:
-                first_matching_event = all_events.get(organization=organization)
-                events.append(first_matching_event.id)
-            except:
-                pass
-        return Event.objects.filter(id__in=events).order_by("start_time")
+        return Event.objects.filter(start_time__gte=date.today()).distinct("organization")
```

Comment:
Flake8 wasn't a fan of the bare `except:` above, but I believe this can be simplified instead.